pkg/services/sqlstore/quota.go

```diff
@@ -255,7 +256,19 @@ func (ss *SQLStore) UpdateUserQuota(ctx context.Context, cmd *models.UpdateUserQ
 func (ss *SQLStore) GetGlobalQuotaByTarget(ctx context.Context, query *models.GetGlobalQuotaByTargetQuery) error {
 	return ss.WithDbSession(ctx, func(sess *DBSession) error {
 		var used int64
-		if query.Target != alertRuleTarget || query.UnifiedAlertingEnabled {
+
+		if query.Target == filesTarget {
```

Comment:
Maybe better to put this under the bottom branch? It already has 

```
if query.Target == dashboardTarget {
				rawSQL += fmt.Sprintf(" WHERE is_folder=%s", dialect.BooleanStr(false))
			}

```

For custom per-target raw SQL. Or maybe I am missing some difference here.